tests/conftest.py

```diff
@@ -36,6 +36,16 @@ def api(constants):
     return Api(constants["API_KEY"])
 
 
+@pytest.fixture()
+def api_with_endpoint_url(constants):
+    return Api(constants["API_KEY"], endpoint_url="https://api.example.com")
+
+
+@pytest.fixture()
+def api_with_endpoint_url_trailing_slash(constants):
+    return Api(constants["API_KEY"], endpoint_url="https://api.example.com/")
```

Comment:
This probably doesn't need to be a fixture; it could just be a one-line test. But it's a good edge case to test!

Reply:
https://github.com/gtalarico/pyairtable/pull/243/commits/5ad5c8012a55d0eef06098f684cc281e64201835

Let me know whether or not I misunderstood your suggestion